packages/rum-core/src/domain/rumEventsCollection/action/getActionNameFromElement.ts

```diff
@@ -1,36 +1,38 @@
 import { safeTruncate } from '@datadog/browser-core'
 
-export function getActionNameFromElement(element: Element): string {
+/**
+ * Get the action name from the attribute 'data-dd-action-name' on the element or any of its parent.
+ * This can be customized by the setting the actionNameAttribute configuration option.
+ */
+const PROGRAMMATIC_ATTRIBUTE = 'data-dd-action-name'
```

Comment:
Why not to put 'data-dd-action-name' as a default config in the buildConfiguration function like so:
```typescript
if ('actionNameAttribute' in userConfiguration) {
  configuration.actionNameAttribute = userConfiguration.actionNameAttribute
} else {
  configuration.actionNameAttribute = 'data-dd-action-name'
}
```

Reply:
IMO the advantage of the current approach it that it keeps all the logic at the same place.

what benefit do you see with your suggestion?

Reply:
I like the idea of having the configuration defined in a single place.
Overriding the output of buildConfiguration(), would mean to lose the control of something we want to centralize.
IMO, it's easier to understand and debug.

Reply:
obsolete with f3c7736